changelogs/fragments/vmware_guest.yaml

```diff
@@ -0,0 +1,4 @@
+---
+
+minor_changes:
+  - Added a timeout parameter for wait_for_ip_address for longer-running tasks
```

Comment:
This change is specific to just one module. In this case we prefix the line with the name of the module, so it should be something like:

`-  wait_for_ip_address_timeout - Added a timeout parameter for ``wait_for_ip_address`` for longer-running tasks`

Reply:
I've amended this now.